src/org/rust/lang/core/psi/util/RustModExtensions.kt

```diff
@@ -7,6 +7,8 @@ import org.rust.lang.core.psi.RustModItem
 
 
 val MOD_RS = "mod.rs"
+val MAIN_RS = "main.rs"
+val LIB_RS = "lib.rs"
```

Comment:
I don't think it is necessary to extract this constants, they are a private implementation detail, unlike `MOD_RS`